### DISCUSSION TITLE
Notifications number is higher because of finished running tasks notification

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#20330] [Q&A] [answered]
You can see in the video that after downloading some maps the notifications number (6) does not reflect the number of notifications that you can see in the overlay (3). It's kinda weird seeing it change upon clicking it.
This does not seem to happen always, but most times from my testing.

Should probably not increment if a running task ends in a completion notification, or decrease if there is none.

https://user-images.githubusercontent.com/53081750/190487455-c57b0e5a-0331-4758-a551-dfb77f0620db.mp4

[database.log](https://github.com/ppy/osu/files/9578229/database.log)
[runtime.log](https://github.com/ppy/osu/files/9578234/runtime.log)

**Comment by peppy** [comment, marked as answer]
This looks like a regression (previously there was special logic in place to make sure the transition from progression to completion notification was handled even when not visible).

I'll take a look into it.

**Reply by peppy** [reply]
https://github.com/ppy/osu/pull/20339
